ingestion/src/metadata/profiler/processor/datalake_sampler.py

```diff
@@ -63,8 +54,9 @@ def get_col_row(self, data_frame):
         cols = []
         rows = []
         cols = data_frame[0].columns.tolist()
+        # Sample Data should not exceed sample limit
         for chunk in data_frame:
-            rows.extend(self._fetch_rows(chunk)[: 100 - len(rows)])
+            rows.extend(self._fetch_rows(chunk)[: self.sample_limit])
```

Comment:
Don't you need to keep the `- len(row)` here so you know how many rows you need to add?

Reply:
as the chunk in itself is 200000 big, we won't need to do that..